### ISSUE TITLE
Implement new app detail view

### ISSUE THREAD

**Issue by k88hudson** [mozilla/webmaker-app#868]
Design: https://redpen.io/wy5af842444b6a2667
( Design ticket: https://github.com/mozilla/webmaker-app/issues/839 )

Note that:

* this should replace the current `detail` view
* the edit app button and app data tab should only appear if the user owns the app

**Comment by gesa** [comment]
The redpen link is to a deleted design. Could I get an updated link @vazquez @k88hudson ?

**Comment by vazquez** [comment]
Hey Aki, the design is not finalized yet, but this is a link of the work in progress. We are close, this will get finalized by tomorrow. https://redpen.io/ck555642cf38b02adb

**Comment by k88hudson** [comment]
@gesa The designs are now finalized in #839! Redpens here:

Admin: https://redpen.io/ck555642cf38b02adb
Consumer: https://redpen.io/vd06771345f2712a65


cc @vazquez

**Comment by gesa** [comment]
Quick q:

Should I be integrating this checkbox in this ticket, or is it addressed elsewhere? (this is gonna be a big PR)
![](http://snaps.akibraun.com/p3fuz.png)

cc @k88hudson @vazquez

**Comment by vazquez** [comment]
Good catch, we will make a separate ticket for this Aki.

**Comment by vazquez** [comment]
Aki I just looked around and there is an existing Issue filed, #865

**Comment by gesa** [comment]
oh! thanks @vazquez! I thought that was about one of these buggers: ![](http://snaps.akibraun.com/jizuv.png)

**Comment by gesa** [comment]
Blocked by #951
